crates/typst/src/symbols/sym.rs

```diff
@@ -486,7 +486,16 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
     refmark: '※',
     servicemark: '℠',
     maltese: '✠',
-    suit: [club: '♣', diamond: '♦', heart: '♥', spade: '♠'],
+    suit: [
+        club.filled: '♣',
+        club.stroked: '♧',
```

Comment:
Maybe it's better to do something like
```suggestion
        club: ['♣', stroked: '♧'],
```

Reply:
I think this would be inconsistent with other symbols.

Reply:
The diff is fine, I believe, since Typst defaults the symbol to the first variant, so writing `club` will still return the `filled` variant.